examples/passport-login/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@loopback/passport-login",
+  "version": "1.0.0",
+  "description": "examples to demonstrate authentication with passport strategies",
+  "main": "index.js",
+  "engines": {
+    "node": ">=10"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "build:watch": "lb-tsc --watch",
+    "clean": "lb-clean dist *.tsbuildinfo package",
+    "lint": "npm run prettier:check && npm run eslint",
+    "lint:fix": "npm run eslint:fix && npm run prettier:fix",
+    "prettier:cli": "lb-prettier \"**/*.ts\" \"**/*.js\"",
+    "prettier:check": "npm run prettier:cli -- -l",
+    "prettier:fix": "npm run prettier:cli -- --write",
+    "eslint": "lb-eslint --report-unused-disable-directives .",
+    "eslint:fix": "npm run eslint -- --fix",
+    "pretest": "npm run clean && npm run build",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "posttest": "npm run lint",
+    "test:dev": "lb-mocha --allow-console-logs dist/__tests__/**/*.js && npm run posttest",
+    "migrate": "node ./dist/migrate",
+    "prestart": "npm run build",
+    "start": "node .",
+    "prepublishOnly": "npm run test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strongloop/loopback-next.git",
+    "directory": "examples/passport-login"
+  },
+  "author": "IBM Corp.",
+  "license": "MIT",
+  "dependencies": {
+    "@loopback/authentication": "^4.1.1",
+    "@loopback/authentication-passport": "^2.0.0",
+    "@loopback/boot": "2.0.2",
+    "@loopback/context": "3.2.0",
```

Comment:
Please use `^` for all `@loopback/*` dependencies.

Reply:
@raymondfeng , agreed, fixed now